doc/changelogs/CHANGELOG_V19.md

```diff
@@ -0,0 +1,209 @@
+# Node.js 19 ChangeLog
+
+<!--lint disable maximum-line-length no-literal-urls prohibited-strings-->
+
+<table>
+<tr>
+<th>Current</th>
+</tr>
+<tr>
+<td>
+<a href="#19.0.0">19.0.0</a><br/>
+</td>
+</tr>
+</table>
+
+* Other Versions
+  * [18.x](CHANGELOG_V18.md)
+  * [17.x](CHANGELOG_V17.md)
+  * [16.x](CHANGELOG_V16.md)
+  * [15.x](CHANGELOG_V15.md)
+  * [14.x](CHANGELOG_V14.md)
+  * [13.x](CHANGELOG_V13.md)
+  * [12.x](CHANGELOG_V12.md)
+  * [11.x](CHANGELOG_V11.md)
+  * [10.x](CHANGELOG_V10.md)
+  * [9.x](CHANGELOG_V9.md)
+  * [8.x](CHANGELOG_V8.md)
+  * [7.x](CHANGELOG_V7.md)
+  * [6.x](CHANGELOG_V6.md)
+  * [5.x](CHANGELOG_V5.md)
+  * [4.x](CHANGELOG_V4.md)
+  * [0.12.x](CHANGELOG_V012.md)
+  * [0.10.x](CHANGELOG_V010.md)
+  * [io.js](CHANGELOG_IOJS.md)
+  * [Archive](CHANGELOG_ARCHIVE.md)
+
+<a id="19.0.0"></a>
+
+## 2022-10-18, Version 19.0.0 (Current), @RafaelGSS and @ruyadorno
+
+Node.js 19 is here! Highlights include the update of the V8 JavaScript engine to 10.8, HTTP(s)/1.1 KeepAlive enabled by default, and ESM Resolution adjusts.
+
+Initially, Node.js 19 will replace Node.js 18 as our ‘Current’ release line. As per the release schedule, Node.js 18 ‘Hydrogen’ will be the 'LTS and Node.js 19 will be ‘Current' release for the next 6 months and then enter ‘Maintenance Mode’ in April 2023.
```

Comment:
I don't think we should mention the 'Maintenance Mode' for an odd-numbered release line here as users may get it confused with 'LTS Maintenance' and expect it to be the full 18 months. 

We could drop this line or just simplify it to say something like:

```suggestion
Node.js 19 will replace Node.js 18 as our ‘Current’ release line when Node.js 18 enters long-term support (LTS) later this month. As per the release schedule, Node.js 19 will be ‘Current' release for the next 6 months, until April 2023.
```